packages/angular_devkit/build_angular/src/utils/normalize-optimization.ts

```diff
@@ -6,13 +6,28 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import { OptimizationClass, OptimizationUnion } from '../browser/schema';
+import { FontsClass, OptimizationClass, OptimizationUnion } from '../browser/schema';
+
+export type NormalizeOptimizationOptions = Required<Omit<OptimizationClass, 'fonts'>> & {
+  fonts: FontsClass,
+};
+
+export function normalizeOptimization(optimization: OptimizationUnion = false): NormalizeOptimizationOptions {
```

Comment:
Not exactly sure what the default value here means - isn't the `default` in the schema `true`?

Reply:
Strictly speaking the default is defined in the schematic which is `false`. But in the typings these are still marked as optional.